src/app/page.js

```diff
@@ -1,3 +1,4 @@
+import Registration from "@/components/Registration/Registration";
```

Comment:
You don't need this import as you don't use this components here

Reply:
Yes, I forgot to delete it.